server/shared/types/dataset.ts

```diff
@@ -61,13 +61,21 @@ type GenomicPositionEntry = {
 */
 
 type Chr2bcffile = { [index: string]: string }
+type bcfMafFile = {
+	bcffile: string // bcf file for only variants, no samples and FORMAT
+	maffile: string // maf file for sample mutations. bcf header contents with FORMAT and list of samples are copied into this maf as headers (any else)
```

Comment:
please fully clarify the maf format

Reply:
Please check if this if enough:

// maf file for sample mutations. bcf header contents with FORMAT and list of samples are copied into this maf as headers followed by the maf header stariting with #chr, pos, ref, alt and sample. Each column after sample corresponds to the information in FORMAT. maf file is gzipped by bgzip and indexed by tabix. The command to index maf file: tabix -c"#" -s 1 -b 1 -e 1 <maf.gz>

Reply:
good please put in the comment